lib/linter-eslint.coffee

```diff
@@ -72,7 +72,7 @@ module.exports =
         # Add showRuleId option
         showRuleId = atom.config.get 'linter-eslint.showRuleIdInMessage'
 
-        if rulesDir and fs.existsSync rulesDir
+        if rulesDir and statSync(rulesDir).isDirectory()
```

Comment:
You have to do:

```
try
  options.rulePaths = [rulesDir] if rulesDir and statSync(rulesDir).isDirectory()
```

Because `statSync(rulesDir)` will throw an error if it doesn't exists...